### ISSUE TITLE
About the way the ChestX-ray8 dataset is divided into the training set and the validation set and the test set.

### ISSUE THREAD

**Issue by jcwang0602** [eraserNut/MedRPG#1]
Hello, this method of yours is a great job. In your paper, you report the performance of MedRPG on the ChestX-ray8 dataset, but do not give a specific split method. How did you divide the 984 images with box annotations in ChestX-ray8?
